### PR TITLE
Add available_targets and ambassadors

### PR DESCRIPTION
**Pull request by colorlace**

### CODE REVIEW

File: app/services/available_targets.rb [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+      end
+    end
+
+  end
+end
+
+
+   # VoteLeader:  room
+   # Show/RequestShare:  room - self
+   # Select Hostage: room - self - tackled
```
Comment (efatsi) [inline app/services/available_targets.rb:31]: looks like you left some shit lying around

**Review by efatsi — CHANGES_REQUESTED** [review]

File: app/services/available_targets.rb (hunk 1 above)
Comment (colorlace) [reply app/services/available_targets.rb]: I done fixed it

**Review by efatsi — APPROVED** [review]
looks real good

File: app/services/available_targets.rb [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+    @room_players = room.players
+    @game = room.game
+  end
+
+  def available_targets
+
+    if action.is_a?(Actions::VoteLeader)
+      return room_players
+    else
+      roombutself = room_players.reject{|p| p==from_player}
```
Comment (efatsi) [inline app/services/available_targets.rb:17]: try
```
room_but_self = room_players - [from_player]
```

File: app/services/available_targets.rb [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+      if action.is_a?(Actions::SelectHostage)
+        return roombutself.reject{|p| p.is_tackled?}
```
Comment (efatsi) [inline app/services/available_targets.rb:19]: this could be 
```rb
roombutself.reject(&:tackled?)
```

**Comment by efatsi** [comment]
Looks like this is already merged, closing PR

**Pull request closed without merging**